### PR TITLE
Add Contributing.md file

### PR DESCRIPTION
**Pull request by Manidipaaa**
Added a Contributing.md file and fixed the grammatical errors. Removed the contents of the readme file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+A new Flutter application.
+
+## Getting Started
+
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+
```
Comment (geekymeeky) [inline CONTRIBUTING.md]: Remove these lines

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+
+For help getting started with Flutter, view our
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
```
Comment (geekymeeky) [inline CONTRIBUTING.md]: Remove these lines

**Review by geekymeeky — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,112 +0,0 @@
...
-
-<p align="center"><img src="https://media.giphy.com/media/5mCQOcUfywmyI/giphy.gif" width=35%></p>
-
-## ✨ Contributors
-Thanks go to these Wonderful People 👨🏻‍💻 🚀 Contributions of any kind are welcome! 
-
-
-<a href="https://github.com/geekymeeky/chitchat/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=geekymeeky/chitchat" />
-</a>
```
Comment (geekymeeky) [inline README.md:100]: Do not delete these lines.

**Review by geekymeeky — CHANGES_REQUESTED** [review]

**Comment by Manidipaaa** [comment]
Updated CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+<p align="center"><img src="https://media.giphy.com/media/5mCQOcUfywmyI/giphy.gif" width=35%></p>
+
+## ✨ Contributors
+All thanks go to these Wonderful People 👨🏻‍💻 🚀 Contributions of any kind are welcome! 
+
+
+<a href="https://github.com/geekymeeky/chitchat/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=geekymeeky/chitchat" />
+</a>
```
Comment (geekymeeky) [inline CONTRIBUTING.md:89]: Remove these lines

File: README.md (hunk 3 above)
Comment (geekymeeky) [inline README.md:100]: Do not delete these lines

**Review by geekymeeky — CHANGES_REQUESTED** [review]
Changes that must be addressed:
- Remove `Contributors` section from `CONTRIBUTING.md`
- Contributors section should be in the Readme file. So, don't delete those markup.

**Comment by geekymeeky** [comment]
@Manidipaaa any update?

**Pull request closed without merging**